src/main/scala/com/databricks/labs/overwatch/pipeline/SnapshotRunner.scala

```diff
@@ -0,0 +1,49 @@
+package com.databricks.labs.overwatch.pipeline
+
+import com.databricks.labs.overwatch.utils.SparkSessionWrapper
+import com.databricks.labs.overwatch.utils.{BadConfigException, SparkSessionWrapper}
+import org.apache.log4j.{Level, Logger}
+
+
+object SnapshotRunner extends SparkSessionWrapper {
+  private val logger: Logger = Logger.getLogger(this.getClass)
+
+  def main(args: Array[String]): Unit = {
+    envInit()
+    if (args.length == 3) { //Snapshot with default pipeline, tablesToExclude and  cloneLevel
```

Comment:
no need to check var args size

Reply:
Done. But I think we need one check for lower limit and upper limit for the mandatory params.